requirements-dev.txt

```diff
@@ -50,6 +52,10 @@ mccabe==0.6.1
     # via flake8
 mypy-extensions==0.4.3
     # via black
+numpy==1.22.2
+    # via opencv-python
+opencv-python==4.5.5.62
+    # via -r requirements-dev.in
```

Comment:
- [ ] Actually, I don't have any test with this yet. I should add one

Reply:
The openCV is for PNGs only and I don't want to add svg to png converters, because they are not the main thing here. So, I removed the package that was needed after all.